src/firefox/lib/glue.js

```diff
@@ -170,6 +180,58 @@ function setUpConnection(freedom, panel, button) {
       });
     }
   });
+
+
+  // Check if user already has a tab open to the uProxy install page.
+  for (var tab of tabs) {
+    if (isInstallPage(tab.url)) {
```

Comment:
I agree that'd be nice. Do you know if there's already an API call we can use to match the `include` value above, i.e.
```javascript
[ self.data.url('*'),
                 "https://www.uproxy.org/*",
                 "https://test-dot-uproxysite.appspot.com/*"]
```
(Note the globs.) If not, do you agree we can punt on this for now rather than having to start implementing our own glob matching?